pre-index.php

```diff
@@ -595,15 +595,15 @@ class="onoffswitch-checkbox"/>
             <center>
                 <button class="settings"
                         onclick="confirm('Are you sure you want to reset settings to default values?') ? (localStorage.clear(), window.location.reload()) : false">
-                    <i class="fa fa-refresh" aria-hidden="true"></i> Reset Settings
+                    <i class="fa fa-refresh" aria-hidden="true"></i> <?php echo i8ln('Reset Settings') ?>
                 </button>
             </center>
         </div>
         <div>
             <center>
                 <button class="settings"
-                        onclick="download('<?= addslashes($title) ?>', JSON.stringify(JSON.stringify(localStorage)))">
```

Comment:
why did you remove addslashes() function?

Reply:
Corrected